tests/dags/test_dags.py

```diff
@@ -0,0 +1,39 @@
+import glob
+import pytest
+from os import path
+from airflow import models
+from airflow.utils.dag_cycle_tester import test_cycle as cycle_test
+
+
+DAG_PATHS = glob.glob(path.join(path.dirname(__file__), "..", "..", "dags", "[!_]*.py"))
```

Comment:
An alternative to using glob with multiple path methods calls would be to use a list comprehension with the standard `pathlib` module:

```python
import pathlib
DAG_PATHS = [p for p in (pathlib.Path(__file__).parents[2]/'dags').glob("[!_]*.py")]
``` 
This might be more readable?